packages/@react-aria/interactions/src/useFocusWithin.ts

```diff
@@ -17,6 +17,7 @@
 
 import {DOMAttributes} from '@react-types/shared';
 import {FocusEvent, useCallback, useRef} from 'react';
+import {getActiveElement, getOwnerDocument} from '@react-aria/utils';
```

Comment:
Can be done later, but would be nice to have a lint rule or something that makes sure that we make use of these instead of using `document.activeElement`